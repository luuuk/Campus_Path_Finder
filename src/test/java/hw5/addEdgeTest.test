# This test is for the addEdge method
# Create a new Graph, g
CreateGraph g

# Add node n1 to g
AddNode g n1

# Add node n2 to g
AddNode g n2

# Add edge from n1 to n2
AddEdge g n1 n2 n1ton2

# Add edge from n2 to n1
AddEdge g n2 n1 n2ton1

# Add edge from n1 to n1
AddEdge g n1 n1 n1ton1