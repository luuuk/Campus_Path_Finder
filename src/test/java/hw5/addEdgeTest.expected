# This test is for the addEdge method
# Create a new Graph, g
created Graph g

# Add a new Node, n1 to g
added node n1 to g

# Add node n2 to g
added node n2 to g

# Add edge from n1 to n2
added edge n1ton2 from n1 to n2 in g

# Add edge from n2 to n1
added edge n2ton1 from n2 to n1 in g

# Add edge from n1 to n1
added edge n1ton1 from n1 to n1 in g